src/layer/BlanketOverlay.js

```diff
@@ -145,20 +149,20 @@ export const BlanketOverlay = Layer.extend({
 	 */
 	_initContainer() {
 		this._container = DomUtil.create('div');
-	},
+	}
 	_destroyContainer() {
 		DomEvent.off(this._container);
 		this._container.remove();
 		delete this._container;
-	},
+	}
 	_resizeContainer() {
 		const p = this.options.padding,
 		    size = this._map.getSize().multiplyBy(1 + p * 2).round();
 		this._container.style.width = `${size.x}px`;
 		this._container.style.height = `${size.y}px`;
 		return size;
-	},
-	_onZoomEnd: Util.falseFn,
-	_onViewReset: Util.falseFn,
-	_onSettled: Util.falseFn,
-});
+	}
+	_onZoomEnd() {}
+	_onViewReset() {}
+	_onSettled() {}
```

Comment:
Those changes were essentials to fix the remaining three tests. Otherwise, somehow, inheritance did not work.